smarts/core/coordinates.py

```diff
@@ -90,12 +90,12 @@ class Point(NamedTuple):
     def from_np_array(cls, np_array: np.ndarray):
         """Factory for constructing a Point object from a numpy array."""
         assert 2 <= len(np.array) <= 3
-        z = np_array[2] if len(np_array) > 2 else 0.0
+        z = np_array[2] if len(np.array) > 2 else 0.0
```

Comment:
Should this be `np.array` or `np_array`?

Reply:
doh!!! yes.  Thanks.
I did the "fix" in reverse (b/c I thought it hadn't been made yet).